com.unity.render-pipelines.core/Runtime/Debugging/Prefabs/Scripts/DebugUIHandlerEnumField.cs

```diff
@@ -8,7 +8,8 @@ namespace UnityEngine.Rendering.UI
     /// </summary>
     public class DebugUIHandlerEnumField : DebugUIHandlerWidget
     {
-        /// <summary>Name of the enum field.</summary>
+        public Text nextButtonText;
+        public Text previousButtonText;
```

Comment:
Documented in my PR: https://github.com/Unity-Technologies/Graphics/commit/a8aac544e1ed8cb07b244197a84178184d0953e8#diff-1cc4af64637a63c015a171cc760f78bf814d502aa0fd24fddb64037b98d9b569

Public because it needs to be visible in editor.

Reply:
Docs added.